medusa/server/web/config/post_processing.py

```diff
@@ -213,19 +220,13 @@ def isNamingValid(pattern=None, multi=None, abd=False, sports=False, anime_type=
             return 'invalid'
 
     @staticmethod
-    def isRarSupported():
-        """
-        Test Packing Support:
-            - Simulating in memory rar extraction on test.rar file
-        """
-
+    def isRarSupported(unrar_tool):
```

Comment:
Not all linux installations have unrar. Can you say with 100% sure ALL distros have?